### PR TITLE
Add IG: Marketplace Product Offer + Checkout feature integration

### PR DESCRIPTION
**Pull request by psturm-spryker**
## PR Description

https://spryker.atlassian.net/browse/MP-3513

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+## Install feature core
+Follow the steps below to install the Marketplace Product Offer + Checkout feature integration feature core.
+
+### Prerequisites
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+|-|-|-|
+| Spryker Core | master | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration)  |
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:17]: Checkout feature should be mentioned as well.

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\CheckoutExtension\Dependency\Plugin\CheckoutPreConditionPluginInterface[]
+     */
+    protected function getCheckoutPreConditions(Container $container)
+    {
+        return [
+            // other plugins ..
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:89]: Please remove this line.

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            new ProductOfferCheckoutPreConditionPlugin(),
+        ];
+    }
+}
+```
+
+---
+**Verification**
+
+Make sure that when refreshing the checkout summary page, after changing `active` or `approved` status of a product offer, the status is reflected accordingly.
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:100]: Please add detailed description where should i go to deactivate/change status of the offer, and more important, what will happen on checkout page.

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+Make sure that the following modules were installed:
+
+| MODULE | EXPECTED DIRECTORY |
+|-|-|
+| CartExtension | spryker/cart-extension |
+| CheckoutExtension | spryker/checkout-extension |
+| Product | spryker/product |
+| ProductOfferExtension | spryker/product-offer-extension |
+| Store | spryker/store |
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:38]: It's fine to get rid of those modules.

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+---
+**Verification**
+
+Make sure that the following modules were installed:
+
+| MODULE | EXPECTED DIRECTORY |
+|-|-|
+| CartExtension | spryker/cart-extension |
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:34]: It's fine to get rid of this modules.

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+| CheckoutExtension | spryker/checkout-extension |
+| Product | spryker/product |
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:36]:
```suggestion
| ProductOffer | spryker/product-offer |
```

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+| CheckoutResponseTransfer | class | Created | src/Generated/Shared/Transfer/CheckoutResponseTransfer |
+| ItemTransfer.merchantReference | property | Created | src/Generated/Shared/Transfer/ItemTransfer |
+
+---
+
+### 3) Configure Checkout Pre Condition Plugins
+
+| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
+|-|-|-|-|
+| ProductOfferCheckoutPreConditionPlugin | Checks if at least one quote item transfer has items with inactive or not approved ProductOffer. |   | Spryker/Zed/ProductOffer/Communication/Plugin/Checkout/ProductOfferCheckoutPreConditionPlugin.php |
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:66]:
```suggestion
| ProductOfferCheckoutPreConditionPlugin | Blocks checkout if at least one quote item transfer has items with inactive or not approved ProductOffer. |   | Spryker/Zed/ProductOffer/Communication/Plugin/Checkout/ProductOfferCheckoutPreConditionPlugin.php |
```

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md (hunk 1 above)
Comment (psturm-spryker) [reply docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md:17]: Is `202001.0` the correct version? Copied it from https://github.com/spryker/spryker.github.io/blob/master/docs/marketplace/dev/feature-integration-guides/marketplace-shipment-checkout-feature-integration.md

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**